atmo/jobs/provisioners.py

```diff
@@ -56,7 +56,10 @@ def run(self, user_email, identifier, emr_release, size,
                 'Name': 'setup-telemetry-spark-job',
                 'ScriptBootstrapAction': {
                     'Path': self.script_uri,
-                    'Args': ['--timeout', str(job_timeout * 60)]
+                    'Args': [
+                        '--timeout', str(job_timeout * 60),
+                        '--email', user_email,
```

Comment:
Yes, @mreid-moz made a great point that people could use it to lazily rely on past state and whatnot in their bucket. We should keep all data files and outputs in scheduled jobs in s3.